org.raceintospace.Raceintospace.yml

```diff
@@ -0,0 +1,77 @@
+# vim: sts=4:
+app-id: org.raceintospace.Raceintospace
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: raceintospace
+rename-desktop-file: raceintospace.desktop
+rename-appdata-file: raceintospace.appdata.xml
+finish-args:
+    - --share=ipc
+    - --socket=x11
+    - --socket=pulseaudio
+    - --persist=.raceintospace
+    # Until support for SDL2, only X11 will work
+    #- --socket=wayland
+modules:
```

Comment:
Not critical, but would be nice to follow https://github.com/flathub/flathub/wiki/YAML-Style-Guide. I forgot about it on previous review.